app/controllers/admin/projects_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Admin::ProjectsController < Admin::BaseController
+  def index
+    @project = Project.new
```

Comment:
Easier to have it there than another link. Also, it matches hwo we have it in category.